src/sweetalert2.scss

```diff
@@ -608,12 +608,12 @@ body {
 .swal2-icon {
   position: relative;
   justify-content: center;
-  width: 80px;
-  height: 80px;
+  width: $swal2-icon-size;
+  height: $swal2-icon-size;
   margin: $swal2-icon-margin;
-  border: 4px solid transparent;
+  border: .25rem solid transparent;
   border-radius: 50%;
-  line-height: 80px;
+  line-height: 5.5rem;
```

Comment:
Set this to `$swal2-icon-size`. Currently, it's misaligned.

Current (5.5rem) | Expected (`$swal2-icon-size`)
---|---
![image](https://user-images.githubusercontent.com/6059356/36655310-88d6a720-1aca-11e8-9ae8-27288576045f.png) | ![image](https://user-images.githubusercontent.com/6059356/36655319-98892788-1aca-11e8-8484-33abb18eb7cf.png)

Reply:
@limonte : My mistake, I was on Firefox and the exclamation mark is too high. On chrome though it's ok. I guess it depends on font used. If I set it to Arial for example it seems ok.
I will set it back to `swal2-icon-size`, and maybe we could investigate on this difference beteween browsers (if you can reproduce the issue).